integration-tests/create-react-app/src/App.percy.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
```

Comment:
Came up with a better solution for this. We now automatically inject a webpack loader that adds an ESLint comment to the beginning of percy snapshot files whitelisting our global variables. So the special eslint-disable comment is no longer necessary!